vowpalwabbit/parse_args.cc

```diff
@@ -1616,6 +1616,12 @@ vw* initialize_with_builder(std::unique_ptr<options_i, options_deleter_type> opt
 
     print_enabled_reductions(all, enabled_reductions);
 
+    if (!all.logger.quiet)
+    {
+      *(all.trace_message) << "Input label = " << to_string(all.example_parser->lbl_parser.label_type) << std::endl;
```

Comment:
we could also print something fancier rather than the explicit type, but at least this gives a clue on what was the computed expected input and what is going to be the output type pred (in theory)